tools/clang/lib/SPIRV/Constant.cpp

```diff
@@ -44,6 +44,18 @@ const Constant *Constant::getFloat32(SPIRVContext &ctx, uint32_t type_id,
   return getUniqueConstant(ctx, c);
 }
 
+const Constant *Constant::getFloat64(SPIRVContext &ctx, uint32_t type_id,
+                                     double value, DecorationSet dec) {
+  struct wideFloat {
```

Comment:
Oh, this depends on the host machines endianness? We need a TODO to mention that here at least.

Reply:
Done